sample/README.md

```diff
@@ -26,9 +26,9 @@ Perform Static Analysis:
 
 We are using this tools to add Static Analysis and Generics to our PHP code:
 
-- [PHPStan](https://phpstan.org/)
-- [Psalm](https://psalm.dev/docs/)
-- @todo Update version of PHP-CS-Fixer to PHP 8.0 `"friendsofphp/php-cs-fixer": "^2.16"`
+-   [PHPStan](https://phpstan.org/)
+-   [Psalm](https://psalm.dev/docs/)
+-   @todo Update version of PHP-CS-Fixer to PHP 8.0 `"friendsofphp/php-cs-fixer": "^2.16"`
```

Comment:
I really hate this from Prettier… why on earth it adds spaces here? xD

Reply:
Agree.